analyzer/src/main/kotlin/managers/PhpComposer.kt

```diff
@@ -307,7 +307,9 @@ class PhpComposer(
         }.toSet()
 
     private fun parseDeclaredLicenses(packageInfo: JsonNode) =
-        packageInfo["license"]?.mapNotNullTo(sortedSetOf()) { it?.textValue() } ?: sortedSetOf<String>()
+        sortedSetOf<String>().also { set ->
+            packageInfo["license"]?.let { set += it.textValue() }
```

Comment:
The license field is an array, you still need to iterate over this.

Reply:
Uh, thanks :eyes: